### PR TITLE
Addig /purgeparticles 

### PR DESCRIPTION
**Pull request by Adamx120**
It's the repairing work of /clearparticles

### CODE REVIEW

**Comment by OnlineRepz** [comment]
Yeah... About that, I think you should check it before making PR...

**Comment by Adamx120** [comment]
But Is it fine?

File: src/me/StevenLawson/TotalFreedomMod/Command_purgeparticles.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+public class Command_purgeparticles extends TFM_Command
+{
+    @Override
+    public boolean run(CommandSender sender, Player sender_p, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
+    {
+        TFM_Util.adminAction(sender.getName(), "Purging all playerparticle data", true);
+        {
+        for (Player player : server.getOnlinePlayers())
+        {
+        server.dispatchCommand(player, "pp clear");
```
Comment (WickedGamingUK) [inline src/me/StevenLawson/TotalFreedomMod/Command_purgeparticles.java]: This should be avoided.

File: src/me/StevenLawson/TotalFreedomMod/Command_purgeparticles.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+@CommandPermissions(level = AdminLevel.SUPER, source = SourceType.BOTH)
+@CommandParameters(description = "Cleaning Particles", usage = "/<command>")
+public class Command_purgeparticles extends TFM_Command
+{
+    @Override
+    public boolean run(CommandSender sender, Player sender_p, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
+    {
+        TFM_Util.adminAction(sender.getName(), "Purging all playerparticle data", true);
+        {
```
Comment (WickedGamingUK) [inline src/me/StevenLawson/TotalFreedomMod/Command_purgeparticles.java]: This should cause an error and will lead to TFM not being able to be compiled.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_clearchat.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+@CommandPermissions(level = AdminLevel.SUPER, source = SourceType.BOTH)
+@CommandParameters(description = "Simply cleanup tool", usage = "/<command>", aliases = "cc")
+public class Command_clearchat extends TFM_Command 
+{        
+    @Override
+    public boolean run(CommandSender sender, Player sender_p, Command cmd, String commandLabel, String[] args, boolean senderIsConsole) {
+        for (int i = 0; i <= 80; i++)
+        {    
+            TFM_Util.bcastMsg("");
```
Comment (WickedGamingUK) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_clearchat.java]: I'm sure there's a better way of clearing chat than this, but whatever.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_clearparticles.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+import org.bukkit.entity.Player;
+
+@CommandPermissions(level = AdminLevel.SUPER, source = SourceType.BOTH)
+@CommandParameters(description = "Particle Cleanup", usage = "/<command>", aliases = "cp")
+public class Command_clearparticles extends TFM_Command 
+{        
+    @Override
+    public boolean run(CommandSender sender, Player sender_p, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
+    {
+        {
```
Comment (WickedGamingUK) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_clearparticles.java]: Again, this (and line 19) should cause TFM to not be compiled.

**Comment by OnlineRepz** [comment]
4 files with same coding, fix it or it will be denied

**Comment by WickedGamingUK** [comment]
Learn Java properly first. There's so many mistakes in this regarding formatting, bad practise and shit that just doesn't work.

**Comment by Adamx120** [comment]
Requesting to delete this PR
[I'll make a new one, with all rules]

**Pull request closed without merging**